sql/src/main/java/io/crate/executor/transport/task/elasticsearch/ESGetTask.java

```diff
@@ -340,6 +363,20 @@ public Object extract(GetResponse response) {
                         return response.getId();
                     }
                 };
+            } else if (field.equals("_raw")) {
+                return new FieldExtractor<GetResponse>() {
+                    @Override
+                    public Object extract(GetResponse response) {
+                        return response.getSourceAsBytesRef().toBytesRef();
+                    }
+                };
+            } else if (field.equals("_doc")) {
+                return new FieldExtractor<GetResponse>() {
+                    @Override
+                    public Object extract(GetResponse response) {
+                        return response.getSource();
+                    }
+                };
```

Comment:
would make sense to put a if field.startswith("_") around all sys col checks in order to speed up the normal case which has no system columns